api/src/main/resources/metadata/Role_Privilege.xml

```diff
@@ -624,6 +624,8 @@
     <privilege  privilege="Manage Procedures"  description="Allows the user to manage procedure orders"  uuid="ce981491-76d8-4925-b8d8-089352538a34"/>
     <privilege  privilege="Manage Billing"  description="Allows the cashier to manage billing requests"  uuid="bfe2218c-86db-49d0-93c7-6012c4953fee"/>
 
+    <privilege  privilege="Manage Patient Billing"  description="Manage Patient Billing"  uuid="0c37f23a-bdc4-434d-b1f1-7af02c3dac64"/>
```

Comment:
@Mwanje whats the difference with the `Manage Billing` on `line 625`

Reply:
@akileng56  This update introduces a sub-link on the patient summary page, allowing billing items to be added directly to a patient's record.